usagov/vote-gov-drupal
# Vote-925 -> State Registration Drop Down Test

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[Vote-925](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-925&assignee=611272e0eb7aef00694b8a97)

## Description (optional)
This PR will add test for the drop-down for state registration.  

> **NOTE**
> This test will click though each option... at the start of the test it will grab the value that is listed in the option.  After that, it will store the value and then click the option.  After it clicks the option it will grab the URL for the page and ensure that the URL contains the value that was stored to make sure that the user went to the correct URL  

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull donw branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:open`
5. select the `state-registration.cy.js` test and verify that is is passing
